src/mongo/models.js

```diff
@@ -12,7 +14,15 @@ const BridgeClaimSchema = new Schema({
     cennznetAddress: String,
 }, { collection: BRIDGE_CLAIM });
 
+const EventProcessedSchema = new Schema({
+    _id: String,
+    eventId: String, // txHash from ethereum bridge contract - deposit
```

Comment:
should be possible to use eventId as the mongo _id

Reply:
ok

Reply:
<img width="1280" alt="Screen Shot 2021-11-15 at 3 32 08 PM" src="https://user-images.githubusercontent.com/29415595/141713052-60c6ccf4-3be7-4d42-bd6b-7c12a1c4433b.png">
I cannot update this _id field as it is https://stackoverflow.com/questions/9694460/difference-between-id-and-id-fields-in-mongodb/9694625#9694625
It is supposed to be auto generated.. so I will keep it as it is

Reply:
_id auto-generates a primary key but you can configure it to use a natural primary key like 'event id' in this case

Reply:
it's better for searching by event id